### PR TITLE
 data/data: remove old unused etcd srvs records 

### PR DESCRIPTION
**Pull request by abhinavdahiya**
This requires: https://github.com/openshift/installer/pull/596; https://github.com/openshift/machine-config-operator/pull/155
/cc @crawford

### CODE REVIEW

**Comment by abhinavdahiya** [comment]
https://github.com/openshift/machine-config-operator/pull/155 merged.

/retest

File: data/data/aws/main.tf [hunk 1]
```diff
@@ -117,14 +117,22 @@ resource "aws_route53_record" "etcd_a_nodes" {
...
-resource "aws_route53_record" "etcd_cluster" {
+resource "aws_route53_record" "etcd_cluster_old_srvs" {
   type    = "SRV"
   ttl     = "60"
   zone_id = "${local.private_zone_id}"
   name    = "_etcd-server-ssl._tcp"
   records = ["${formatlist("0 10 2380 %s", aws_route53_record.etcd_a_nodes.*.fqdn)}"]
 }
 
+resource "aws_route53_record" "etcd_cluster" {
```
Comment (wking) [inline data/data/aws/main.tf]: Accidentally restoring these?

Comment (wking) [reply data/data/aws/main.tf]: Never mind, I was just confused by the #596 commit being shown here.

**Comment by abhinavdahiya** [comment]
```console
2018/11/02 03:26:38 Copying artifacts from release-latest into /logs/artifacts/release-latest
info: Using registry public hostname registry.svc.ci.openshift.org
info: Found 79 images in image stream
info: Manifests will be extracted to /tmp/release-image-0.0.1-2018-11-02T032624Z400021104
warning: Could not load current user information: user: unknown userid 1089000000
Unable to connect to the server: net/http: TLS handshake timeout
2018/11/02 03:26:38 Container release in pod release-latest failed, exit code 1, reason Error
```

/retest

**Comment by abhinavdahiya** [comment]
Requires https://github.com/openshift/machine-config-operator/pull/156
/hold

**Comment by wking** [comment]
openshift/machine-config-operator#156 landed.

/hold cancel
/retest

**Pull request merged**